depends/toolchain.cmake.in

```diff
@@ -131,3 +135,7 @@ endif()
 if(NOT WITH_USDT AND "@no_usdt@" STREQUAL "1")
   set(WITH_USDT OFF CACHE STRING "")
 endif()
+
+if(NOT HARDENING AND "@no_harden@" STREQUAL "1")
+  set(HARDENING OFF CACHE STRING "")
```

Comment:
I was expecting to see this set hardening to true if not @no_harden@, but it's already the default in the main build. So it's doing the same thing as our config.site which makes sense.